scss/_patterns_pricing-block.scss

```diff
@@ -0,0 +1,48 @@
+@import 'settings';
+
+@mixin vf-p-pricing-block {
+  .p-pricing-block,
+  .p-pricing-block--25-75,
+  .p-pricing-block--50-50 {
+    @extend %vf-grid-row-equal-height;
+
+    .p-pricing-block__tier {
+      &:before {
+        border-top: 1px solid $colors--theme--border-default;
+        content: '';
+        margin-bottom: calc($spv--small - 1px);
+        width: calc(100% + map-get($grid-gutter-widths, default));
+      }
+
+      &:last-child::before {
+        width: 100%;
+      }
+
+      @media (max-width: $threshold-4-6-col) {
+        grid-column: span 4;
```

Comment:
This is the small size colspan - I wonder if it should be the default case instead of inside a media query? Then we can use media queries for medium and large

Also, again, it would be nice if this was related to the grid settings:

`span #{$grid-8-columns-small}`